### ISSUE TITLE
Support non- `/usr/local/` paths for Homebrew prefix

### ISSUE THREAD

**Issue by JamesMcMahon** [KryptCo/homebrew-tap#5]
I use a custom path for my Homebrew prefix. When installing kr through the tap I needed to go in and manually clean up my `.ssh/config` and replace `/usr/local/` with my custom prefix.

Would it be possible to support this out of the box?

**Comment by kcking** [comment]
Have you exported the `HOMEBREW_PREFIX` environment variable? `kr` reads this during `kr pair` and uses it instead of `/usr/local` if it is set (the exact logic is here: https://github.com/kryptco/kr/blob/2f43a8bb0ce036420580320558e7ff9f8b065bc1/kr/kr_darwin.go#L17)

**Comment by JamesMcMahon** [comment]
No, now that I look at it I didn't have `HOMEBREW_PREFIX` set. I may have missed but looking through the docs I don't see that ever called out as something that to be set to apply a non-standard prefix. My concern would be that no one else would have this set since it's really not called out by Homebrew.

Just spit balling here, but is it possible to resolve `brew --prefix` to determine the prefix?
